pom.xml

```diff
@@ -338,6 +343,16 @@
             <artifactId>junit</artifactId>
             <version>4.12</version>
             <scope>compile</scope>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
```

Comment:
hmm, asta la ce e folosit?

Reply:
Am avut eroarea aia cu servlet si una din rezolvari era sa adauge in maven, dar nu a mers si nu stersesem eu partea aia. O s-o sterg acum.